frontend/src/metabase/dashboard/actions/actions.unit.spec.js

```diff
@@ -153,158 +151,4 @@ describe("dashboard actions", () => {
       });
     });
   });
-
-  describe("fetchDashboardParameterValuesWithCache", () => {
```

Comment:
Added a e2e test for parameter caching, and it has already found issues that were not spotted with the previous implementation.